.changeset/bold-rockets-thank.md

```
@@ -0,0 +1,7 @@
+---
+'@mastra/inngest': major
+---
+
+Inngest durable agents now reject response-only recovery configuration before dispatch because it requires live worker state.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)